connection.go

```diff
@@ -21,14 +20,14 @@ type Conn struct {
 	// after Ready is closed.
 	MountError error
 
-	// File handle for kernel communication. Only safe to access if
-	// rio or wio is held.
+	// File handle for kernel communication.
 	dev *os.File
-	wio sync.RWMutex
```

Comment:
As far as I can tell, reads are atomic and ingest the entire message. And since there's no bookkeeping here, there's nothing much to protect. Am I missing something major/obscure?